addons/hr_holidays/models/hr_leave.py

```diff
@@ -262,10 +262,10 @@ def _compute_dashboard_warning_message(self):
         ])
         for holiday in self:
             conflicting_holidays = all_leaves.filtered_domain([
-                ('employee_id', '=', holiday.employee_id.id),
+                ('employee_id', 'in', holiday.employee_id.ids),
                 ('date_from', '<', holiday.date_to),
                 ('date_to', '>', holiday.date_from),
-                ('id', '!=', holiday.id),
+                ('id', 'not in', holiday.ids),
```

Comment:
I didn't really investigate your PR, but why this change? It's making it much more ambiguous to read IMO

Reply:
Because during onchange, NewId is used which results in a warning in Domains.
FYI we are discussing whether we should accept `('id', 'in', holiday)` and similar.